### PR TITLE
DOC: release notes

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

It's time for a release

### CODE REVIEW

File: docs/source/releases.rst [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+
+v2.6.0 (2020-05-21)
+===================
+
+Features
+--------
+- ``happi`` entry points have been moved to this library for proper
+  modularization.
+- Area detectors embedded inside of larger devices have been made
+  considerably smaller to speed up embedded applications, for
```
Comment (klauer) [inline docs/source/releases.rst:13]: 'embedded applications'?

File: docs/source/releases.rst [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+  example in ``typhos``.
+
+Bugfixes
+--------
+- Provide ``FakePytmcSignal`` for testing in external libraries. This
+  fixes issues with fake devices not working if they contain ``PytmcSignal``
+  instances outside of the ``pcdsdevices`` testing suite.
+- Fix various issues related to moving to ``ophyd`` ``v1.5.0``.
+- This library is now portable to win32.
```
Comment (klauer) [inline docs/source/releases.rst:22]:
```suggestion
- This library is now importable on win32.
```

**Comment by ZLLentz** [comment]
The py37 failed... with no code changes. Running again for science, but I'm going to merge and tag regardless because we're deploying py36.

**Comment by ZLLentz** [comment]
Oh you, race condition. It happened when checking the `umv` utility function, which timed itself out in the status API.

**Pull request merged**